design/capm3-remediation-controller-proposal.md

```diff
@@ -104,24 +104,28 @@ Metal3RemediationTemplate.
 #### Remediation Controller
 
 * MRC watches for the presence of Metal3Remediation CR.
-* Based on the remediation strategy defined in ```.spec.strategy``` in
+* Based on the remediation strategy defined in ```.spec.strategy.type``` in
   Metal3Remediation, RC uses BMO APIs to get hosts back into a healthy or
   manageable state.
 * RC uses ```.status.phase``` to save the states of the remediation. Available
-  states are running, rebooting, power off, failed.
+  states are running, waiting, deleting machine.
 * After RC have finished its remediation, it will wait for the Metal3Remediation
   CR to be removed. (When using CAPI MachineHealthCheck controller,
   MHC will noticed the Node becomes healthy and deletes the instantiated
   MachineRemediation CR.).
-* If RCs timeout for Node to become healthy expires, it annotates BareMetalHost
-  with ```capi.metal3.io/unhealthy``` annotation, sets BMH Online field
-  to False and removes the Machine object.
+* If RCs timeout for Node to become healthy expires, it sets
+  `capi.MachineOwnerRemediatedCondition` to False on Machine object
+  to start deletion of the unhealthy Machine and the corresponding Metal3Remediation.
+* If RCs timeout for Node to become healthy expires, it annotates BareMetalHost with
+  `capi.metal3.io/unhealthy` annotation, sets BMH Online field to False and
+  removes the Machine object.
```

Comment:
So after machine deletion, we expect a new machine to be created.
does this new machine will be able to use that powered off host in case other hosts are not available? will BMO change the online state?

Reply:
I am not quite sure how BMO handles this? But if it is not able to power on the host that should be issued and take care in BMO.